static/js/components/PhotometryTable.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useSelector } from "react-redux";
+import Dialog from "@material-ui/core/Dialog";
+import DialogContent from "@material-ui/core/DialogContent";
+import Slide from "@material-ui/core/Slide";
+import CloseIcon from "@material-ui/icons/Close";
+import IconButton from "@material-ui/core/IconButton";
+import Tooltip from "@material-ui/core/Tooltip";
+
+import MUIDataTable from "mui-datatables";
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  // eslint-disable-next-line
+  return <Slide direction="up" ref={ref} {...props} />;
+});
+
+const PhotometryTable = ({ obj_id, open, onClose }) => {
+  const { photometry } = useSelector((state) => state);
+  let bodyContent = null;
+
+  if (!Object.keys(photometry).includes(obj_id)) {
+    bodyContent = <p>Loading...</p>;
+  } else {
+    const data = photometry[obj_id];
+    if (data.length === 0) {
+      bodyContent = <p>Source has no photometry.</p>;
+    } else {
+      const keys = Object.keys(data[0]).filter((key) => key !== "groups");
+      const columns = keys.map((key) => ({
+        name: key,
+        options: { filter: false },
+      }));
+      const formattedData = data.map((dataRow) =>
+        keys.map((key) => dataRow[key])
+      );
```

Comment:
```suggestion
```
I don't think you need to format it like this. It should work if you just pass `data` to the MUIDataTable, right? Unless there's another reason you're formatting it here that I'm missing.

Reply:
Although actually you'll probably need to keep some form of this in order to add the sig figs and/or units that Josh suggested (or just use customBodyRenderLite() functions in the column options instead).

Reply:
data is an array of objects, MUIdatatables needs an array of arrays

Reply:
Uhhh no it can definitely be an array of objects - I've been using them in other components and the example data on the main readme for the package is also an array of objects. 
![Screenshot 2020-11-30 200719](https://user-images.githubusercontent.com/17696889/100684141-beb29000-3347-11eb-9249-995ff12f9ba5.png)
https://github.com/gregnb/mui-datatables